libs/zedUpload/awsutil/s3sync.go

```diff
@@ -8,10 +8,13 @@ import (
 	"compress/gzip"
 	"fmt"
 	"io"
+	"log"
```

Comment:
It makes more sense to have the caller pass in some logger object by defining an log(ger) interface with Info* and Error* functions. That way we can have downloader pass in its log variable which is setup with logrus to do the right thing.

Reply:
Modified